core-api/src/main/java/com/optimizely/ab/config/audience/OrCondition.java

```diff
@@ -44,15 +44,15 @@ public OrCondition(@Nonnull List<Condition> conditions) {
     // false or null is null
     // false or false is false
     // null or null is null
-    public @Nullable Boolean evaluate(ProjectConfig config, Map<String, ?> attributes) {
+    public @Nullable
+    Boolean evaluate(ProjectConfig config, Map<String, ?> attributes) {
```

Comment:
Same as other comments about annotation placement for return types. This is formatting is worse.